iModelCore/ECDb/Tests/NonPublished/ECSqlPrepareTests.cpp

```diff
@@ -1149,6 +1149,14 @@ TEST_F(ECSqlSelectPrepareTests, OrderBy)
     EXPECT_EQ(ECSqlStatus::Success, Prepare("SELECT I, S FROM ecsql.P ORDER BY MyPSA.RelECClassId"));
     EXPECT_EQ(ECSqlStatus::Success, Prepare("select I FROM ecsql.PSA ORDER BY NULLIF(I,123)"));
     EXPECT_EQ(ECSqlStatus::Success, Prepare("select I FROM ecsql.PSA ORDER BY COALESCE(I,L)"));
+    //adding macro to test nulls first and last changes

```

Comment:
@spyboy01  I do not see you have pushed any test with actual data to show if FIRST LAST order is null correctly and native sql generated includes the first/last keyword.

Reply:
@khanaffan I have written a separate test, but build was failing due to some error in return type status. That's why I didn't commit the code. Now that the build went fine, I will push the changes by today's EOD.

Reply:
CI job failing on the test you added.
```
[ RUN      ] ECSqlStatementTestFixture.NullsOrdering
D:\vsts_a\85\bbsrc\imodel-native\iModelCore\ECDb\Tests\NonPublished\ECSqlStatementTests.cpp(9176): error: Expected equality of these values:
  firstName
    Which is: "Albert"
  "nullptr"
```

Reply:
I have modified the code and it is working fine now.